backend/schemas/event.ts

```diff
@@ -1,53 +1,56 @@
 const event = {
     name: "event",
-    title: "Events",
+    title: "Event",
     type: "document",
     fields: [
         {
             name: "title",
-            title: "Title",
+            title: "Title på Event ",
+            description:"Tittel til event. F.eks 'PR VERV'",
             type: "string",
         },
         {
             name: "description",
-            title: "Description (Kort forklaring til event kort)",
+            title: " Beskrivelse av Event",
             type: "text",
+            description : "En kort beskrivelse av Event(maks 150 ord)",
             validation: (Rule: { max: (arg0: number) => any }) => Rule.max(150),
         },
         {
             name: "datetime",
-            title: "Date and Time",
+            title: "Dato og tidspunkt på event",
             type: "datetime",
         },
         {
             name: "slug",
-            title: "Slug",
+            title: "URL-mer leselig lenke(slug)",
             type: "slug",
```

Comment:
Slug is hard to explain well, but I think you did a good job here! Maybe we'll get some feedback on this one.